public/js/load_more.js

```diff
@@ -13,7 +13,11 @@ $(function() {
             dataType: 'html',
             success : function(code_html, statut)
             {
-                $(code_html).appendTo(".trick-list");
+                if(code_html == "") {
+                    $('.load').hide();
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/F-Jean/SnowTricks/pullRequest?prid=8521799)
```suggestion
                    $(".load").hide();
```